experiments/spotify/1_no_code_consume_source/README.md

```diff
@@ -0,0 +1,36 @@
+# Consume a ready source without Python code
+We can load a source from the command line. Imagine this:
+
+```
+pip install python-dlt
+pip install python-dlt-contrib
+
+dlt init
+
+vi .dlt/secrets
+
+dlt source info dlt.contrib.sources.spotify
+
+> displays the source docstring and parameters to be passed - all from function signature!
+
+dlt load --to bigquery --dataset spotify dlt.contrib.sources.spotify --data-dir my_data --tables library,history
```

Comment:
@adrianbr is a CLI like this really that much better for folks than a short Python file? This command is quite unwieldy

Reply:
@adrianbr are you sure this is our guy? someone that cannot save and run python script will not be able to run pipelines. he would be able only to experiment in notebook but then what's next?